artifacts/chart/Chart.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: v2
+name: karmada
+description: A Helm chart for karmada
+
+# A chart can be either an 'application' or a 'library' chart.
+#
+# Application charts are a collection of templates that can be packaged into versioned archives
+# to be deployed.
+#
+# Library charts provide useful utilities or functions for the chart developer. They're included as
+# a dependency of application charts to inject those utilities and functions into the rendering
+# pipeline. Library charts do not define any templates and therefore cannot be deployed.
+type: application
+
```

Comment:
Should we add a [kubeVersion Field](https://helm.sh/docs/topics/charts/#the-kubeversion-field), like `>=1.16.0`? Since we are using `apiextensions.k8s.io/v1` which serves start [Kubernetes v1.16](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.16.md).

Reply:
Yes, I will do it.